resources/lib/ps_vue.py

```diff
@@ -545,6 +549,7 @@ def create_device_id():
 def utc_to_local(utc_dt):
     # get integer timestamp to avoid precision lost
     timestamp = calendar.timegm(utc_dt.timetuple())
+    #local_dt = datetime(1970, 1, 1) + timedelta(seconds=timestamp)
```

Comment:
There's an issue with next airings and old shows which cause this error:
local_dt = datetime.fromtimestamp(timestamp)
ValueError: timestamp out of range for platform localtime()/gmtime() function

Which led me to change the code as described in this "fix":
https://stackoverflow.com/questions/36179914/timestamp-out-of-range-for-platform-localtime-gmtime-function

You can recreate the issue by selecting a channel with old show, usually Turner Classic Movies is a perfect candidate.

Reply:
Ok, I'll keep reading up on the timedelta. Maybe the timestamp variable for the seconds is off in some cases. I'll make another commit if I find a solution

Reply:
I subtracted 25200 from the seconds and that corrected the airing times on my test box. 
local_dt = datetime(1970, 1, 1) + timedelta(seconds=timestamp-25200)

If you want to test it and see if it subtracts 7hrs from your times or leaves them the same. My guess is that daylight savings is affecting different builds of Kodi time.